File: explorer/src/main/java/com/duongame/manager/PositionManager.java
package com.duongame.manager;

import java.util.HashMap;

/**
 * Created by namjungsoo on 2016-11-20.
 */

public class PositionManager {
    private static HashMap<String, Integer> positionMap = new HashMap<String, Integer>();
    private static HashMap<String, Integer> topMap = new HashMap<String, Integer>();

    public static void setPosition(String path, int position) {
        positionMap.put(path, position);
    }

    public static int getPosition(String path) {
        if (positionMap.containsKey(path)) {
            return positionMap.get(path);
        }
        return 0;
    }

    public static void setTop(String path, int top) {
        topMap.put(path, top);
    }

    public static int getTop(String path) {
        if (topMap.containsKey(path)) {
            return topMap.get(path);
        }
        return 0;
    }
}

File: Explorer/src/main/java/com/duongame/manager/PositionManager.kt
package com.duongame.manager

import com.duongame.manager.PositionManager
import java.util.HashMap

/**
 * Created by namjungsoo on 2016-11-20.
 */
object PositionManager {
    private val positionMap = HashMap<String, Int>()
    private val topMap = HashMap<String, Int>()

    fun setPosition(path: String, position: Int) {
        positionMap[path] = position
    }

    fun getPosition(path: String): Int {
        return if (positionMap.containsKey(path)) {
            positionMap[path]!!
        } else 0
    }

    fun setTop(path: String, top: Int) {
        topMap[path] = top
    }

    fun getTop(path: String): Int {
        return if (topMap.containsKey(path)) {
            topMap[path]!!
        } else 0
    }
}